hardware-and-software-requirements.md

```diff
@@ -8,13 +8,13 @@ aliases: ['/docs/dev/hardware-and-software-requirements/','/docs/dev/how-to/depl
 
 As an open source distributed NewSQL database with high performance, TiDB can be deployed in the Intel architecture server, ARM architecture server, and major virtualization environments and runs well. TiDB supports most of the major hardware networks and Linux operating systems.
 
-## Linux OS version requirements
+## OS and platform requirements
 
 | Linux OS        | Version      |
 | :-----------------------:| :----------: |
-| Red Hat Enterprise Linux | 7.3 or later 7.x releases |
-| CentOS                   | 7.3 or later 7.x releases |
-| Oracle Enterprise Linux  | 7.3 or later 7.x releases |
+| Red Hat Enterprise Linux | 7.3 or later 7.x versions, 8.4 or later 8.x versions |
+| CentOS                   | 7.3 or later 7.x versions, 8.4 or later 8.x versions |
```

Comment:
```suggestion
| CentOS Linux         | 7.3 or later 7.x versions |
| CentOS Stream      | 8                                    |
```

Reply:
@dveeden According to the QA team, the Stream 8 is verified only in TiDB v6.1.x and not in v6.2 or v6.3. Therefore, I add it in https://github.com/pingcap/docs/pull/10595/files#diff-d794b834a513233d979186ee859a99d50e5ea7679598b84aeaab0aea12739a7bR22. PTAL

Reply:
```suggestion
| CentOS                   | 7.3 or later 7.x versions |
```